### PR TITLE
Update deploy in Makefile

### PR DESCRIPTION
**Pull request by hongkailiu**
To keep up with the example in operator-sdk:

- Add a namespace like [the example](https://github.com/operator-framework/operator-sdk/blob/e95abdbd5ccb7ca0fd586e0c6f578e491b0a025b/testdata/go/v4/memcached-operator/config/manager/manager.yaml#L2).
- Modify `deploy` like [the example](https://github.com/operator-framework/operator-sdk/blob/e95abdbd5ccb7ca0fd586e0c6f578e491b0a025b/testdata/go/v4/memcached-operator/Makefile#L194).

### CODE REVIEW

File: config/manager/manager.yaml [hunk 1]
```diff
@@ -1,3 +1,10 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  labels:
+    app.kubernetes.io/managed-by: kustomize
+  name: openshift-updateservice
```
[the remark this answers is not in the document]
Comment (hongkailiu) [reply config/manager/manager.yaml:6]: I think there is a way.
May I handle it with another PR?
(The namespace is not introduced now.)

[the remark this answers is not in the document]
Comment (petr-muller) [reply config/manager/manager.yaml:6]: A followup sounds good to me

**Review by petr-muller — APPROVED** [review]

**Pull request merged**